eZ/Publish/Core/FieldType/Tests/DateAndTimeTest.php

```diff
@@ -229,15 +229,15 @@ public function testFromHash($inputHash, $expectedResult)
 
         $actualResult = $fieldType->fromHash($inputHash);
 
-        // Tests may run slowly. Allow 20 seconds margin of error.
+        // Tests may run slowly. Allow 60 seconds margin of error.
         $this->assertGreaterThanOrEqual(
             $expectedResult,
             $actualResult,
             'fromHash() method did not create expected result.'
         );
         if ($expectedResult->value !== null) {
             $this->assertLessThan(
-                $expectedResult->value->getTimestamp() + 20,
+                $expectedResult->value->getTimestamp() + 60,
```

Comment:
tests aren't _that_ slow, if you still get more then 20 seconds of deviations there more likely is a bug in here somewhere